app/controllers/api/status_controller.rb

```diff
@@ -23,9 +25,33 @@ def check_legacy
   end
 
   def check_new
+    @includes = %i[repository]
+
     # This forces serialization with Oj
     # TODO: Investigate calling Oj.mimic_JSON on initializationh to swap universally
     serializer = OptimizedProjectSerializer.new(projects, project_names, internal_api_key?)
```

Comment:
Oj was doing a bad job serializing TimeWithZone and I didn't want to fight too hard